conf/config.yaml

```diff
@@ -39,6 +39,7 @@ apisix:
   #   - 127.0.0.0/24              # If we don't set any IP list, then any IP access is allowed by default.
   #   - "::/64"
   # port_admin: 9180              # use a separate port
+  token_admin: 33926bc55db5e2c3   # default token when use API to call for Admin API
```

Comment:
1. we need add doc in the admin API
2. we need to change examples of admin API
3. better more strong default token
4. It is strongly recommended ithat users modify this default value. How about write in doc and comment?

Reply:
all done, please take a look when you have time ^_^